content/developer/reference/extract_api.rst

```diff
@@ -465,19 +468,47 @@ represents an invoice line. Each value follows a
     "invoice_lines": [
         {
             "description": feature_result,
-            "discount": feature_result,
-            "product": feature_result,
             "quantity": feature_result,
             "subtotal": feature_result,
             "total": feature_result,
             "taxes": feature_result,
             "total": feature_result,
-            "unit": feature_result,
             "unit_price": feature_result
         },
         ...
     ]
 
+Bank statements
+~~~~~~~~~~~~~~~
+
+The following table gives a list of all the fields that are extracted from bank statements.
+
++-------------------------+------------------------------------------------------------------------+
+| Feature name            | Specificities                                                          |
++=========================+========================================================================+
+| ``balance_start``       | ``content`` is a float                                                 |
++-------------------------+------------------------------------------------------------------------+
+| ``balance_end``         | ``content`` is a float                                                 |
++-------------------------+------------------------------------------------------------------------+
+| ``date``                | ``content`` is a string                                                |
++-------------------------+------------------------------------------------------------------------+
+
+``feature_result`` for the ``bank_statement_lines`` feature
+***********************************************************
+
+It is returned as a list of dictionaries where each dictionary represents a bank statement line.
+Each value follows a :ref:`latestextract_api/get_result/feature_result` structure.
+
+.. code-block:: js
+
+    "bank_statement_lines": [
+        {
+            "amount": feature_result,
+            "description": feature_result,
+            "date": feature_result,
```

Comment:
It's actually not a `feature_result` for bank statements, we return the value directly.